Review.md

```diff
@@ -1,6 +1,20 @@
 # Please answer the following questions
 
 1.  In Jest, what are the differences between `describe()` and `it()` globals, and what are good uses for them?
+
+     -it() is the method that you use for each individual test.
+    -describe() is the method that you use for testing components
+
 2.  What is the point of `Test Driven Development`? What do you think about this approach?
+
+     -to make sure that everything works as it should for the user.
+
 3.  What are `mocks`? What are a good use cases for them?
+
+    -mocks are how you test the functions and how many times they are called
```

Comment:
mock is fake representation of a component. We use is to control the output provided  by the dependency.